File: generated-sources/scala-gatling/mojang-sessions/src/gatling/scala/org/openapitools/client/model/PlayerTexture.scala

package org.openapitools.client.model


case class PlayerTexture (
    _SKIN: Option[PlayerSkinURL],
    _CAPE: Option[PlayerTextureURL]
)
object PlayerTexture {
    def toStringBody(var_SKIN: Object, var_CAPE: Object) =
        s"""
        | {
        | "SKIN":$var_SKIN,"CAPE":$var_CAPE
        | }
        """.stripMargin
}

File: generated-sources/kotlin-spring/mojang-sessions/src/main/kotlin/org/openapitools/model/PlayerTexture.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.PlayerSkinURL
import org.openapitools.model.PlayerTextureURL
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * Provide links to the player's skin and cape
 * @param SKIN 
 * @param CAPE 
 */
data class PlayerTexture (

        @JsonProperty("SKIN") val SKIN: PlayerSkinURL? = null,

        @JsonProperty("CAPE") val CAPE: PlayerTextureURL? = null
) {

}

